watertap/costing/watertap_costing_package.py

```diff
@@ -793,6 +855,66 @@ def cost_membrane(blk, membrane_cost, factor_membrane_replacement):
     )
 
 
+def cost_electrodialysis_stack(
+    blk,
+    membrane_cost,
+    spacer_cost,
+    membrane_replacement_factor,
+    electrode_cost,
+    electrode_replacement_factor,
+):
+    """
+    Generic function for costing the stack in an electrodialysis unit.
+    Assumes the unit_model has a `cell_pair_num`, `cell_width`, and `cell_length`
+    set of variables used to size the total membrane area.
+
+    Args:
+        membrane_cost - The total cost of the CEM and AEM per cell pair in currency per area
+
+        spacer_cost - The total cost of the spacers per cell pair in currency per area
+
+        membrane_replacement_factor - Replacement factor for membranes and spacers
+                                      [fraction of membranes/spacers replaced/year]
+
+        electrode_cost - The total cost of electrodes in a given stack in currency per area
+
+        electrode_replacement_factor - Replacement factor for electrodes
+                                        [fraction of electrodes replaced/year]
+    """
+    make_capital_cost_var(blk)
+    make_fixed_operating_cost_var(blk)
+
+    blk.membrane_cost = pyo.Expression(expr=membrane_cost)
+    blk.membrane_replacement_factor = pyo.Expression(expr=membrane_replacement_factor)
+    blk.spacer_cost = pyo.Expression(expr=spacer_cost)
+    blk.electrode_cost = pyo.Expression(expr=electrode_cost)
+    blk.electrode_replacement_factor = pyo.Expression(expr=electrode_replacement_factor)
+
+    blk.capital_cost_constraint = pyo.Constraint(
+        expr=blk.capital_cost
+        == (blk.membrane_cost + blk.spacer_cost)
+        * (
+            blk.unit_model.cell_pair_num

```

Comment:
we do not need a `2 * ` here because the multiplier has been included in the unit price, correct?  Fine in this way, but I think it may cause confusion for users. Especially for spacers, they actually need to give a doubled unit price as the unit price when using this model.  For membrane, they'll need to add up unit prices of cem and aem.

Reply:
Yeah, I see what you are saying. I think it probably makes sense as well to divide it up by an actual per unit cost. I remember at a prior meeting it was decided not to break it up, but we can change it.